src/main/java/com/minecolonies/coremod/colony/buildings/BuildingLumberjack.java

```diff
@@ -160,8 +249,12 @@ public String getJobName()
     public static class View extends AbstractBuildingWorker.View
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 70 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)